transmit-lib/src/cmr/transmit/urs.clj

```diff
@@ -59,6 +63,24 @@
        response
        (not (nil? response))))))
 
+(defn get-user-info
+  "Returns URS info associated with a username"
+  [context user]
+  (let [{:keys [status body]} (request-with-auth context {:url-fn #(user-info-url % user)
+                                                          :method :get
+                                                          :raw? true})]
+   (when-not (= 200 status)
+     (errors/internal-error!
```

Comment:
We should log the error message coming back from EDL in the log.

Reply:
👍